package.json

```diff
@@ -31,8 +31,8 @@
     "immutable": "^3.7.4"
   },
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react": "^0.14.0 || 15.0.0-rc.1",
```

Comment:
Can we do `^0.14.0 || ^15.0.0-rc` which will catch the RCs and all subsequent 15.x stable versions?